src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/ngapacts/DcirNgapActExtractor.scala

```diff
@@ -70,12 +71,28 @@ class DcirNgapActExtractor(ngapActsConfig: NgapActConfig) extends DcirExtractor[
     import input.sqlContext.implicits._
 
     {
-      if (ngapActsConfig.acts_categories.isEmpty) {
+      if (ngapActsConfig.actsCategories.isEmpty) {
         input.filter(isInExtractorScope _)
       }
       else {
         input.filter(isInExtractorScope _).filter(isInStudy(codes) _)
       }
     }.flatMap(builder _).distinct()
   }
+
+  def dcirIsInCategory(
```

Comment:
This is better. 

Just a remark on the formatting. It should be like this:
```scala
def dcirIsInCategory(
    categories: List[NgapActClassConfig],
    row: Row): Boolean = {
```

Reply:
Done